app/src/main/java/com/github/featuredetectandroid/ui/GrayscaleViewModel.kt

```diff
@@ -0,0 +1,40 @@
+package com.github.featuredetectandroid.ui
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.asImageBitmap
+import androidx.lifecycle.ViewModel
+import com.github.featuredetectandroid.utils.grayscaleByteArrayToBitmap
+
+class GrayscaleViewModel : ViewModel() {
+    private var grayscaleByteArray by mutableStateOf(byteArrayOf())
+    private var width by mutableStateOf(0)
+    private var height by mutableStateOf(0)
+
+    @Suppress("FunctionNaming")
```

Comment:
> Is this required for some code analysis tool? Android Studio seems to have no problems with the special composables naming.

It required for detekt. I planned to add it to the CI, but I'm not sure how useful it will be.

Reply:
It'll be useful in case you forget to run it or somebody else (me, for example) commits